bindings/c/test/mako/mako.c

```diff
@@ -1901,7 +1942,7 @@ void print_stats(mako_args_t* args, mako_stats_t* stats, struct timespec* now, s
 	uint64_t totalxacts = 0;
 	static uint64_t conflicts_prev = 0;
 	uint64_t conflicts = 0;
-	double durationns = (now->tv_sec - prev->tv_sec) * 1000000000.0 + (now->tv_nsec - prev->tv_nsec);
+	double duration = (now->tv_sec - prev->tv_sec) * 1000000000.0 + (now->tv_nsec - prev->tv_nsec);
```

Comment:
durationns might be a better name since it's in nano seconds.

Reply:
+1. Maybe `duration_ns` is more intuitive?